Samples/Windowing/cs-winforms-unpackaged/winforms_unpackaged_app.csproj

```diff
@@ -11,8 +11,8 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22000.196" />
-		<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0-preview3" />
+		<PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22499-preview" />
```

Comment:
Is there a reason for using a preview version of this package? If not we should revert to 22000.196